elastic/apm-server
# Enable transaction aggregation by default

## Motivation/summary

Enable by default so the UI can set the metrics-based mode to auto or on by default.
We still don't drop unsampled transactions by default.

## Checklist

~- [ ] Documentation has been updated~ there are currently no docs for this feature.

## How to test these changes

1. Run apm-server with default config
2. Send some transactions
3. Wait a minute or so
4. Check for transaction metrics docs (should contain `transaction.duration.histogram`)

## Related issues

Closes #4816
